egs2/aishell/asr1/conf/train_asr_streaming_transformer2.yaml

```diff
@@ -0,0 +1,72 @@
+# network architecture
```

Comment:
ditto.
What is the difference between transformer.yaml  and transformer2.yaml ?